client/my-sites/media-library/list-no-content.jsx

```diff
@@ -21,6 +21,12 @@ export default React.createClass( {
 
 	getLabel() {
 		switch ( this.props.filter ) {
+			case 'this-post':
+				return this.translate( 'There are no media uploaded to this post.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?